org.mwc.debrief.core/src/org/mwc/debrief/core/wizards/CSVExportPage3.java

```diff
@@ -114,8 +118,16 @@ public void widgetSelected(final SelectionEvent e)
 
   private void addPurposeField(final Composite contents)
   {

```

Comment:
@ayeshare - the validation methods for Purpose and Distribution are almost identical.  Could you have a go at refactoring out a common method that they both call?

I'm happy show you how I'd do it, if it seems challenging.

Reply:
done.

Reply:
Yes, I like it.